docs/l2_system_contracts/system_contracts_bootloader_description.md

```diff
@@ -315,7 +315,7 @@ The batch information slots [are used at the beginning of the batch](../../syste
 
 - `[266757..1626756]` – slots where the final batch pubdata is supplied to be verified by the [L2DAValidator](../settlement_contracts/data_availability/custom_da.md).
 
-But briefly, this space is used for the calldata to the L1Messenger’s `publishPubdataAndClearState` function, which accepts the address of the L2DAValidator as well as the pubdata for it to check. The L2DAValidator is a contract that is responsible to ensure efficiency [when handling pubdata](../settlement_contracts/data_availability/custom_da.md). Typically, the calldata `L2DAValidator` would include uncompressed preimages for bytecodes, L2->L1 messages, L2->L1 logs, etc as their compressed counterparts. However, the exact implementation may vary across various ZK chains.
+But briefly, this space is used for the calldata to the L1Messenger’s `publishPubdataAndClearState` function, which accepts the option of DA pubdata commitment scheme as well as the pubdata for it to check. It is expected that input includes uncompressed preimages for bytecodes, L2->L1 messages, L2->L1 logs, etc as their compressed counterparts. Different pubdata commitment schemes can be used by different L1 DA validators.
```

Comment:
This is v29, doc changes should not happen here as we are in the process of migrating. Is it possible to port these changes to 
https://github.com/matter-labs/zksync-era/tree/kl/interop-docs

Reply:
CI doesn't like dead links. So we will need to remove this files completely then